src/components/Faq/FaqAccordion.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { MinusIcon, AddIcon } from '@chakra-ui/icons'
+import {
+  Accordion,
+  AccordionItem,
+  Text,
+  AccordionButton,
+  Box,
+  AccordionPanel,
+  Flex,
+  useColorModeValue,
+  Icon,
+} from '@chakra-ui/react'
+
+const SingleAccordion = ({
+  header,
+  body,
+}: {
+  header: string
+  body: string
+}) => {
+  const accordionBg = useColorModeValue('#ffffff', '#2D3748')
+  const accordionShadow = useColorModeValue('0px 0px 20px 11px #80808012', '')
+  return (
+    <Flex direction="column">
+      <Accordion defaultIndex={[1]} allowMultiple my={{ base: 5 }}>
+        <AccordionItem
+          borderRadius="5"
+          py={{ base: 5 }}
+          bg={accordionBg}
+          border="none"
+          boxShadow={accordionShadow}
```

Comment:
For better performance, rather than use the color mode hook:

```suggestion
          boxShadow="0px 0px 20px 11px #80808012"
          _dark={{ boxShadow: 'none' }}
```

Reply:
To avoid applying a shadow when on dark mode, that's why I did that. 
Does it affect performance to use `useColorModeValue`